Makefile

```diff
@@ -16,3 +16,32 @@ undeploy-camunda:
 
 teardown:
 	kind delete cluster --name hackdays
+
+### Demo stuff
+
+build-demo-starter:
+	docker build -t "gcr.io/zeebe-io/starter:autoscaling-demo" -f demo/starter.Dockerfile ./demo/
```

Comment:
Should we build these within this current repository? gcr.io/zeebe-io is private right?

Reply:
It's private, yes. I can give you read access, many SREs have only read access, after all. I'm also fine just making you an honorary Zeebe dev (so you have some limited access to our GCP project, like other Zeebe devs).

Reply:
We can push them here too, but last time it was sort of a pain to associate multiple containers to your repo, no? If you can figure out the easy way to do that, then sure :D

Reply:
> honorary Zeebe dev
I take that any day! 😄 Then we can skip that part for now and leave it like this. 

I think it shouldn't be too hard to push two images to the repo - I do so with our camunda-operator & kubectl-camunda-cli. But I don't want to waste time right now on figuring that out.